### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
Fixes misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/dioptra/actions/runs/10251861216#summary-28360865831 

The action will report that the changes in this PR would make it happy: https://github.com/jsoref/dioptra/actions/runs/10251861455#summary-28360868175

Note: the work here was done against the default branch but rebased cleanly against `dev` per https://github.com/usnistgov/dioptra/blob/e878f3d4167a4c1a48c2cdc7c4e416f47c7ed026/CONTRIBUTING.md#development-workflow

### CODE REVIEW

File: src/frontend/src/views/CreateJob.vue [hunk 1]
```diff
@@ -160,7 +160,7 @@
   const parameters = ref([])
   const computedValue = computed(() => {
     let output = {}
-    if(parameters.value.legnth === 0) return output
+    if(parameters.value.length === 0) return output
```
Comment (jsoref) [inline src/frontend/src/views/CreateJob.vue:163]: this is notable

File: tests/unit/restapi/lib/actions.py [hunk 2]
```diff
@@ -752,7 +752,7 @@ def remove_tag(
     resource_id: int,
     tag_id: int,
 ) -> TestResponse:
-    """Remove tag tag from the resource with the provided unique ID.
+    """Remove tag from the resource with the provided unique ID.
```
Comment (jsoref) [inline tests/unit/restapi/lib/actions.py:755]: initially i worried about this change, but there isn't actually an argument named `tag`, so there's nothing to worry about

**Review by jkglasbrenner — CHANGES_REQUESTED** [review]
Thank you for submitting a contribution to Dioptra! Fixing these spelling errors is useful, and the team wants to accept these changes. Before we can do that, we have two requests:

1. I marked the small subset of changes that modify code and aren't covered by our current tests. Please split what I've marked into a second PR and undo those changes in this PR. That way we've separated documentation changes from code fixes, and we can check that the code changes don't lead to unexpected behavior elsewhere. 
2. [Please read the statement at the top of the CONTRIBUTORS.md file](https://github.com/usnistgov/dioptra/blob/19553d8613ef0c876ce6650e82c4e49eecc3239b/CONTRIBUTORS.md). If you agree to it, please add your GitHub username to the bottom of the file (you only need to do this in one of the PRs). We can only accept contributions from PR authors that agree to the CONTRIBUTORS statement.

   https://github.com/usnistgov/dioptra/blob/19553d8613ef0c876ce6650e82c4e49eecc3239b/CONTRIBUTORS.md?plain=1#L1-L20

Let us know if you have any questions, thank you!

**Review by jkglasbrenner — APPROVED** [review]
LGTM, thank you for adding your username to the CONTRIBUTORS.md file!

**Pull request merged**